src/OpenColorIO/ops/fixedfunction/FixedFunctionOpCPU.cpp

```diff
@@ -653,6 +686,147 @@ void Renderer_ACES_DarkToDim10_Fwd::apply(const void * inImg, void * outImg, lon
     }
 }
 
+Renderer_ACES_GamutMap13_Fwd::Renderer_ACES_GamutMap13_Fwd(ConstFixedFunctionOpDataRcPtr & data)
+    :   OpCPU()
+{
+    m_limCyan = (float)data->getParams()[0];
+    m_limMagenta = (float)data->getParams()[1];
+    m_limYellow = (float)data->getParams()[2];
+    m_thrCyan = (float)data->getParams()[3];
+    m_thrMagenta = (float)data->getParams()[4];
+    m_thrYellow = (float)data->getParams()[5];
+    m_power = (float)data->getParams()[6];
+
+    m_thrCyan = std::min(0.9999f, m_thrCyan);
+    m_thrMagenta = std::min(0.9999f, m_thrMagenta);
+    m_thrYellow = std::min(0.9999f, m_thrYellow);
+
+    // Precompute scale factor for y = 1 intersect
+    auto f_scale = [this](float lim, float thr) {
+        return (lim - thr) / powf(powf((1.0f - thr) / (lim - thr), -m_power) - 1.0f, 1.0f / m_power);
+    };
+    m_scaleCyan = f_scale(m_limCyan, m_thrCyan);
+    m_scaleMagenta = f_scale(m_limMagenta, m_thrMagenta);
+    m_scaleYellow = f_scale(m_limYellow, m_thrYellow);
+}
+
+__inline float Renderer_ACES_GamutMap13_Fwd::compress(float dist, float lim, float thr, float scale, bool invert) const
+{
+    float comprDist;
+
+    // No compression below threshold
+    if (dist < thr)
+    {
+        comprDist = dist;
+    }
+    else
+    {
+        // Disable compression, avoid nan
+        if (lim < 1.0001f)
+        {
+            return dist;
+        }
+
+        // Scale factor for y = 1 intersect
+        const float scl = scale;
+        // Normalize distance outside threshold by scale factor
+        const float nd = (dist - thr) / scl;
+        const float p = powf(nd, m_power);
+
+        // Compress
+        if (!invert)
+        {
+            comprDist = thr + scl * nd / (powf(1.0f + p, 1.0f / m_power));
+        }
+        // Uncompress
+        else
+        {
+            // Avoid singularity
+            if (dist > (thr + scl))
+            {
+                comprDist = dist;
+            }
+            else
+            {
+                comprDist = thr + scl * powf(-(p / (p - 1.0f)), 1.0f / m_power);
+            }
+        }
+    }
+
+    return comprDist;
+}
+
+__inline float Renderer_ACES_GamutMap13_Fwd::gamut_map(float val, float ach, float lim, float thr, float scale, bool invert) const
+{
+    // Distance from the achromatic axis, aka inverse RGB ratios
+    float dist;
+    if (ach == 0.0f)
```

Comment:
The code formatting is to always have `{` and `}` even for a single line statement.

Reply:
Fixed.